dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -272,6 +273,9 @@ def tests(
     os.environ["RUN_TESTS"] = "true"
     if test_type:
         os.environ["TEST_TYPE"] = test_type
+        if "[" and not test_type.startswith("Providers"):
```

Comment:
``if "["``? looks like a typo

Reply:
Yep. It was supposed to be "safety check" so that "[" is only used in providers :)